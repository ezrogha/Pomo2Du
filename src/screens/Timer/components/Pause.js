import React from 'react'
import { Line, Rect } from 'react-native-svg';

export default ({ pause }) => (
    <>
        <Rect x="180" y="240" width="35" height="45" fill="white" onPress={pause} />
        <Line x1="185" y1="240" x2="185" y2="285" stroke="#009DDD" strokeWidth="5" />
        <Line x1="210" y1="240" x2="210" y2="285" stroke="#009DDD" strokeWidth="5" />
    </>
)
