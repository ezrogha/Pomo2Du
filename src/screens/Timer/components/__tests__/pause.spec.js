import React from 'react'

import Pause from '../Pause';
import renderer from 'react-test-renderer'

describe("Testing Pause Component", () => {
    test("renders correctly", () => {
        const tree = renderer.create(<Pause />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
