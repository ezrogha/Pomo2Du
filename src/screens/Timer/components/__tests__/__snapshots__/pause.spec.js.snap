// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Pause Component should render correctly 1`] = `
Array [
  <RNSVGRect
    fill={
      Array [
        0,
        4294967295,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="45"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "fill",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="35"
    x="180"
    y="240"
  />,
  <RNSVGLine
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={
      Array [
        0,
        4278230493,
      ]
    }
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth="5"
    vectorEffect={0}
    x={0}
    x1="185"
    x2="185"
    y={0}
    y1="240"
    y2="285"
  />,
  <RNSVGLine
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "stroke",
        "strokeWidth",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={
      Array [
        0,
        4278230493,
      ]
    }
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth="5"
    vectorEffect={0}
    x={0}
    x1="210"
    x2="210"
    y={0}
    y1="240"
    y2="285"
  />,
]
`;
